platform/vs/docker-sonic-vs/start.sh

```diff
@@ -7,6 +7,21 @@ HWSKU=Force10-S6000
 
 ln -sf /usr/share/sonic/device/$PLATFORM/$HWSKU /usr/share/sonic/hwsku
 
+pushd /usr/share/sonic/hwsku
+
+[ -f lanemap.ini.orig ] || cp lanemap.ini lanemap.ini.orig
+for p in $(ip link show | grep -oE "eth[0-9]+" | grep -v eth0); do
+    grep ^$p: lanemap.ini.orig
+done > lanemap.ini
+
+[ -f port_config.ini.orig ] || cp port_config.ini port_config.ini.orig
+grep ^# port_config.ini.orig > port_config.ini
+for lanes in $(awk -F ':' '{print $2}' lanemap.ini); do
+    grep -E "\s$lanes\s" port_config.ini.orig
+done >> port_config.ini
+
+popd
```

Comment:
I think it would help to have a comment for this section, I don't think it's clear what this step is doing without context.